server/routes/updateGoal/updateGoalFormValidator.test.ts

```diff
@@ -129,7 +129,7 @@ describe('updateGoalFormValidator', () => {
     const errors = validateUpdateGoalForm(form)
 
     // Then
-    expect(errors).toEqual([{ href: '#steps[0].title', text: 'some-title-error' }])
+    expect(errors).toEqual([{ href: '#steps[0][title]', text: 'some-title-error' }])
```

Comment:
I gave you the wrong steer earlier re: form fields for an array.
I told you it was `steps[1].title` .... its actually `steps[0][title]` (array is zero indexed, and we need to use bracket notation rather than dot notation)